packages/account-v2/src/modules/FinancialAssessmentFields/FinancialAssessmentFields.tsx

```diff
@@ -29,9 +29,9 @@ export const FinancialAssessmentFields = () => {
     const { setFieldValue, values } = formik;
     const { employmentStatus, occupation } = values;
 
-    const hideOccupation = shouldHideOccupation(employmentStatus || '');
+    const hideOccupation = shouldHideOccupation(employmentStatus);
 
-    const filteredOccupationList = useMemo(() => filterOccupationList(employmentStatus || ''), [employmentStatus]);
+    const filteredOccupationList = useMemo(() => filterOccupationList(employmentStatus), [employmentStatus]);
```

Comment:
FormDropDownField is not wrapped with memo() so there is no sense to use useMemo here....

Reply:
useMemo is there to prevent the filterOccupationList being called when the FinancialAssessmentFields re-render. I am not sure how FormDropDownField is affects it.

Reply:
alright, there is not so much calculations, and this memoization does not prevent rerendering of child component.
the main sense of memoization is to avoid rerendering, function call is fast, but rerendering is slow. so no reason to add memo (at least if there is an array for example of thousands of elements and we filter it :) )
of coarse it is better to measure the performance, but I think here is just adds additional code, rather than useful.
how do you think?

Reply:
Updated.